cli-sandbox/README.md

```diff
@@ -0,0 +1,71 @@
+# Proposed CLI design
+
+This code demonstrates the mechanics of implementing a CLI with multiple levels of subcommand nesting.
+The `cli/` directory here would be placed in the sourmash root directory.
+
+Note that the actual arguments are incomplete for the examples below, and incorrect for all the commands/subcommands not shown below.
+Fixing all of that will be a tedious copy/paste job.
```

Comment:
Reiterating: I haven't actually filled in most of the arguments. For the most part, what's in place now are placeholders.